backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -144,7 +144,7 @@ public Optional<OrganizationRoleClaims> createUser(
             .stream();
     var qResults =
         _client.listGroups(generateGroupOrgPrefix(organizationExternalId), null, null).stream();
-    var orgGroups = Stream.concat(searchResults, qResults).distinct().collect(Collectors.toList());
+    var orgGroups = Stream.concat(searchResults, qResults).distinct().toList();
```

Comment:
for curiosity's sake: is there something that changed that allows us just to make a list off of the initial concat here rather than using a collector?

Reply:
This was a sonarcloud minor issue that was flagged with the file. The ability to call `.toList()` was introduced to in [java 16](https://www.baeldung.com/java-16-new-features#add-streamtolist-method-jdk-8180352)!